src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -85,6 +85,8 @@ namespace OpenMS
     defaults_.setValue("strict", "true", "Whether to error (true) or skip (false) if a transition in a transition group does not have a corresponding chromatogram.", {"advanced"});
     defaults_.setValidStrings("strict", {"true","false"});
     defaults_.setValue("use_ms1_ion_mobility", "true", "Performs ion mobility extraction in MS1. Set to false if MS1 spectra do not contain ion mobility", {"advanced"});
+    defaults_.setValue("apply_im_peak_picking", "true", "Whether to apply peaking picking on the ion mobilograms.", {"advanced"});
```

Comment:
additional information could help users to judge the impact. E.g. also when it is enabled and how it interplays with use_ms1_ion_mobility